daemon/config/config.go

```diff
@@ -15,39 +15,42 @@ type Config struct {
 	NetworkConfg network.Config
 
 	// Server listening address.
-	Listen []string
+	Listen []string `json:"listen,omitempty"`
 
 	// ListenCRI is the listening address which serves CRI.
-	ListenCRI string
+	ListenCRI string `json:"listen-cri,omitempty"`
 
 	// Debug refers to the log mode.
-	Debug bool
+	Debug bool `json:"debug,omitempty"`
 
 	// ContainerdAddr refers to the unix socket path of containerd.
-	ContainerdAddr string
+	ContainerdAddr string `json:"containerd,omitempty"`
 
 	// DefaultRegistry is daemon's default registry which is to pull/push/search images.
 	DefaultRegistry string
 
 	// Home directory.
-	HomeDir string
+	HomeDir string `json:"home-dir,omitempty"`
 
 	// ContainerdPath is the absolute path of containerd binary,
 	// /usr/local/bin is the default.
-	ContainerdPath string
+	ContainerdPath string `json:"containerd-path"`
 
 	// TLS configuration
 	TLS utils.TLSConfig
 
 	// Default OCI Runtime
-	DefaultRuntime string
+	DefaultRuntime string `json:"default-runtime,omitempty"`
 
 	// Enable lxcfs
-	IsLxcfsEnabled bool
+	IsLxcfsEnabled bool `json:"enable-lxcfs,omitempty"`
 
 	// LxcfsBinPath is the absolute path of lxcfs binary
-	LxcfsBinPath string
+	LxcfsBinPath string `json:"lxcfs,omitempty"`
```

Comment:
Can json name `lxcfs` be more specific ?

Reply:
the name need to match flag name ` flagSet.StringVar(&cfg.LxcfsBinPath, "lxcfs", "/usr/local/bin/lxcfs", "Specify the path of lxcfs binary")`